libs/ui-watt/src/lib/components/checkbox/watt-checkbox.component.ts

```diff
@@ -14,103 +14,68 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, forwardRef, ViewEncapsulation } from '@angular/core';
 import {
-  ControlValueAccessor,
-  UntypedFormControl,
-  NG_VALUE_ACCESSOR,
-  ReactiveFormsModule,
-} from '@angular/forms';
-import {
-  MatLegacyCheckboxChange as MatCheckboxChange,
-  MatLegacyCheckboxModule as MatCheckboxModule,
-} from '@angular/material/legacy-checkbox';
-
-const customValueAccessor = {
-  multi: true,
-  provide: NG_VALUE_ACCESSOR,
-  useExisting: forwardRef(() => WattCheckboxComponent),
-};
-
-const selector = 'watt-checkbox';
+  Component,
+  ElementRef,
+  HostBinding,
+  ViewEncapsulation,
+  forwardRef,
+  inject,
+} from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, FormsModule } from '@angular/forms';
 
 @Component({
+  selector: 'watt-checkbox',
+  styleUrls: ['./watt-checkbox.component.scss'],
+  template: `<label>
+    <input
+      [(ngModel)]="model"
+      [disabled]="isDisabled"
+      (ngModelChange)="onChange($event)"
+      type="checkbox"
+    />
+    <ng-content />
+  </label>`,
+  standalone: true,
```

Comment:
question: Material's checkbox offered "indeterminate" state. Can we implement that here as well?

From Figma 

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/e5f6923f-9a6b-4ec8-adba-35293d40e25d)